### ISSUE TITLE
Row is covered by horizontal scrollbar when using scrollToRow api

### ISSUE THREAD

**Issue by tibbus** [schrodinger/fixed-data-table-2#346]
When you use scrollToRow api, if the table has a horizontal scrollbar, the row scrolled to is covered by the horizontal scrollbar.

## Expected Behavior
The scrolled to row should be visible on the table, not covered by the horizontal scrollbar.

## Current Behavior
The scrolled to row is barely visible on the table, 50% of it is covered by the horizontal scrollbar.

## Steps to Reproduce (for bugs)
```
<Table rowsCount={40} rowHeight={30} headerHeight={50} width={350} height={400} scrollToRow={20}>
  <Column header={<Cell>Name</Cell>}
    width={200}
    cell={(props) => {
      return <Cell {...props}>{props.rowIndex}</Cell>;
    }}
  />
  <Column header={<Cell>Age</Cell>}
    width={200}
    cell={(props) => {
      return <Cell {...props}>{props.rowIndex}</Cell>;
    }}
  />
</Table>
```

## Context
I'm trying to scroll to a row using the scrollToRow api while having a horizontal scrollbar for the table.

## Your Environment
* Version used: 0.8.14
* Browser Name and version: Chrome 69.0.3497.81 (Official Build) (64-bit)
* Operating System and version (desktop or mobile): Windows 10
* Picture with the issue : https://i.imgur.com/sCQRFt5.jpg

**Comment by wcjordan** [comment]
Could you try the v1 beta?  I think this may be fixed there as we've built out a much improved virtual scroll system.

**Comment by tibbus** [comment]
@wcjordan  How can I access the v1 beta ? I don't see it on the release page.

**Comment by wcjordan** [comment]
yarn add fixed-data-table-2@beta or npm install fixed-data-table-2@beta

**Comment by tibbus** [comment]
@wcjordan Thanks for the help, in the beta version is fixed and it works great.
I see that the latest beta version was released a year ago while the public versions has some releases this year.

Do you have any idea when the beta version will be released as public ? Or are there some functionalities in the public one and not in the beta version ?

**Comment by wcjordan** [comment]
No plans to release the beta right now.  I don't know of any issues with it and everyone who's used it has been happy.  But it hasn't had any QA attention, and we don't use it internally at Schrodinger as we haven't seen the issues which motivated it (yet).

Right now there's no active development on FDT2.  We just shepherd bug fixes and aim for stability.  Given the limited resources and investment, there's not a large motivation to release the new beta and address bugs which might come from it.

Wrt to functionalities not on beta, there have been some bug fixes and small improvements (mostly related to mobile) which haven't been upmerged, but otherwise it's up to date.

**Comment by romadenysyuk** [comment]
Hi. I also have the same issue with the version 0.8.15. Is there a fix for this? I tried v1 beta and this bug is indeed fixed there but there are too many new ones that were not present in 0.8.15.

**Comment by wcjordan** [comment]
I don't think the architecture for 0.8 allows for a fix to this, but I'd consider a PR.  You may be better off filing the issues you see on v1, and we can look at addressing them.

**Comment by pradeepnschrodinger** [comment]
Fixed in beta, and no longer reproducable with the [v1.0.0](https://github.com/schrodinger/fixed-data-table-2/releases/tag/v1.0.0) release.
